docs/developer/devicesimulator.md

```diff
@@ -0,0 +1,55 @@
+# Device Simulator
+
+`Devicesimulator` is used to simulate the load generated by devices.
+
+### Setup
+
+Copy the necessary setup files to run a `devicesimulator`.
+
+    mkdir -p ~/.flightctl/certs/ ~/.config/flightctl/
+    cp bin/agent/etc/flightctl/certs/* ~/.flightctl/certs/
+    cp bin/agent/etc/flightctl/config.yaml ~/.flightctl/agent.yaml
```

Comment:
This assumes a locally running API server and will likely also go away soon. Would it make sense to provide the instructions how to log in and download the agent config as described in the "Getting Started" of the user docs instead? I.e. something like:

```
flightctl login https://api.flightctl.127.0.0.1.nip.io/ --web --insecure-skip-tls-verify
flightctl certificate request --cert-type=enrollment --name=client-enrollment --expiration=365d --output-format=embedded > config.yaml
```

Reply:
That's the exact point: one locally running API server (it will not go away soon) and many devicesimulator hosts.
All this is for a QE to run a **10,000 devices** against a **single server**.

Reply:
We should ensure best practices for the user and dev, I agree with Frank. While we can copy certs technically performing these steps programmatically would require ssh auth etc while the later can all be done on the remote host.